### ISSUE TITLE
Add hint to simplify builder statements

### ISSUE THREAD

**Issue by nimakro** [edvin/tornadofx-idea-plugin#30]
If a builder only has one statement like:
`
button("button") {
   action { println("Hello World") } 
}
`
Then the plugin should show a hint to simplified it to this:
`
button("button").action { println("Hello World") } 
`

**Comment by edvin** [comment]
This can be problematic, because if something inside the action lambda is accessing the button via `this`, that code would fail, and even worse, it could end up targeting another component. Something to be aware of at least. Not saying we should add the hint though, as most of the time, this caveat won't be the case.

**Comment by nimakro** [comment]
That is right the same is true for `runAsyncWithProgress` because it would then replace the parent with a progress indicator and not the node we want to show the progress indicator in.

But I don't see why I can't extract that information from the AST. I will give it a shot and see how complicated it will be to implement it.

**Comment by edvin** [comment]
That would be extremely cool :)
